src/daemon/rpc_command_executor.cpp

```diff
@@ -175,14 +175,24 @@ bool t_rpc_command_executor::print_peer_list() {
     }
   }
 
-  for (auto & peer : res.white_list)
+  if (white)
   {
-    print_peer("white", peer);
+    auto peer = res.white_list.cbegin();
+    const auto end = limit ? peer + std::min(limit, res.white_list.size()) : res.white_list.cend();
```

Comment:
Windows will likely throw a fit on this std::min, as limit is uint64_t, and it wants same length size args, and I think size_t must be 32 bit there. So make limit size_t since it's a size.

Reply:
Fixed